php classes (#391)
TYPO3-Documentation/TYPO3CMS-Reference-TCA: Documentation/ColumnsConfig/Type/Passthrough/Index.rst

Characteristics of `passthrough`:

> * A value sent to the 

Reviewer: DataHandler is missing now?

> * A value sent to the `DataHandler`
>   is just kept as is and put into the database field. Default `FormEngine`
>   however never sends values.

Reviewer 2: ```suggestion
* A value sent to the :php:`DataHandler` is kept as is and put into the database field. However, the default TYPO3 backend forms never send data for a `passthrough` field.
```

> * Unlike none, type passthrough must have a database field.

Reviewer 2: ```suggestion
* Unlike the field type `none`, the field type `passthrough` must have a database field.
```

> * `FormEngine` does not render anything for passthrough types by default. But it can be combined with a custom

Reviewer 2: ```suggestion
* The TYPO3 backend forms do not render anything for `passthrough` fields by default. But they can be combined with a custom
```

>   renderType to make it render something. A user type is better suited for such use cases, though.

Reviewer 2: ```suggestion
  `renderType` to display something. A field of type `user` is better suited for such use cases, though.
```

> * Type passthrough field values are usually not rendered at other places in the backend.

Reviewer 2: ```suggestion
* Values of `passthrough` fields are usually not rendered at other places in the backend.
```

* Field updates by the `DataHandler` get logged and the history/undo function will work with such values.

The `passthrough` field can be useful, if:

* A field needs no rendering, but some data handling using hooks of the `DataHandler`.
* The passthrough type is used by TYPO3 core to handle meta data on records rows that are not shown as fields
  if editing records and only have data logic attached to it. An example is the `l18n_diffsource` field whose
  data is rendered differently in FormEngine at other places if editing a record but still updated and handled
  by the `DataHandler`.

> Typical usages of type passthrough is a field that only needs data evaluation on the `DataHandler` side, but

Reviewer 2: ```suggestion
Typical usages of the field type `passthrough` is a field that only needs data evaluation on the `DataHandler` side, but
```
